src/traces/heatmap/make_bound_array.js

```diff
@@ -0,0 +1,78 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Registry = require('../../registry');
+
+module.exports = function makeBoundArray(trace, arrayIn, v0In, dvIn, numbricks, ax) {
+    var arrayOut = [],
+        isContour = Registry.traceIs(trace, 'contour'),
```

Comment:
Agreed. If it's alright, I wanted to keep this PR (almost) entirely just trivial file reorganization so that small but meaningful changes to get too lost in gigantic line-count PRs. In other words, ideally I'd like to push that change off until subsequent PRs that continue to aim for small changes.